### PR TITLE
Chromedriver validations

### LINKED ISSUE #1687: Do not working on Windows 10
## Expected Behavior
Start working 
## Current Behavior
Give an error message
## Possible Solution (optional)

Cookie file not found, creating cookie...

Error: INFO [2018-03-19 16:29:02] [williansalceda] Session started - 2018-03-19 16:29:02
 WARNING [2018-03-19 16:29:02] [williansalceda] Sorry, Record Activity is not working on Windows. We're working to fix this soon!
 Traceback (most recent call last):
 File "quickstart.py", line 5, in <module>
 session.login()
 File "C:\Users\BoneS\Desktop\InstaPy-master\instapy\instapy.py", line 264, in login
 self.bypass_suspicious_attempt):
 File "C:\Users\BoneS\Desktop\InstaPy-master\instapy\login_util.py", line 133, in login_user
 "//select[@class='_fsoey']/option[text()='English']").click()
 File "C:\Users\BoneS\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webelement.py", line 80, in click
 self._execute(Command.CLICK_ELEMENT)
 File "C:\Users\BoneS\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webelement.py", line 628, in _execute
 return self._parent.execute(command, params)
 File "C:\Users\BoneS\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 312, in execute
 self.error_handler.check_response(response)
 File "C:\Users\BoneS\AppData\Local\Programs\Python\Python36\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
 raise exception_class(message, screen, stacktrace)
 selenium.common.exceptions.WebDriverException: Message: unknown error: call function result missing 'value'
 (Session info: chrome=65.0.3325.162)
 (Driver info: chromedriver=2.34.522940 (1a76f96f66e3ca7b8e57d503b4dd3bccfba87af1),platform=Windows NT 10.0.16299 x86_64)

### PR DESCRIPTION
**Pull request by Tjorriemorrie**
Fixes #1687 (and many others). Hopefully the error message will provide the user the information to update the chromedriver instead of raising issues.

### CODE REVIEW

File: instapy/settings.py [hunk 1]
```diff
@@ -8,3 +8,4 @@ class Settings:
     log_location = os.path.join(BASE_DIR, 'logs')
     database_location = os.path.join(BASE_DIR, 'db', 'instapy.db')
     chromedriver_location = os.path.join(BASE_DIR, 'assets', 'chromedriver')
+    chromedriver_min_version = 2.36
```
Comment (joanroig) [inline instapy/settings.py:11]: I can't get this version on my Raspberry Pi 2, running `apt-get install chromedriver` installs chromedriver 2.33 instead :S

Comment (CharlesCCC) [reply instapy/settings.py:11]: try manually download it through `wget` and move it to the location.

Comment (joanroig) [reply instapy/settings.py:11]: Yes I've already tried it but it appears as an executable instead of a shared library

**Comment by Tjorriemorrie** [comment]
Ref #395

**Comment by joanroig** [comment]
Hi again,

As said, there is no updated version of chromedriver for armhf devices, so the Chrome guide is no longer usable now:
https://github.com/timgrossmann/InstaPy/blob/master/docs/How_to_Raspberry.md

After following the guide step by step on a Raspberry pi2:

screen shot 2018-03-24 at 09 16 01

**Comment by FreakyBoy** [comment]
@joanroig Yes I have the same problem... 
chromedriver 2.36+ is only available on 64bit architecture for linux... so with this update all 32bit system can not use IntsaPy...

**Comment by Tjorriemorrie** [comment]
@joanroig @FreakyBoy  use `git checkout 0666bb8` till above issue is merged.

**Pull request merged**